### ISSUE TITLE
Fancybox Question

### ISSUE THREAD

**Issue by gbrandallen** [stevenschobert/instafeed.js#307]
Great app btw. Really like its simplicity and flexibility.

Quick question. Is there a way you are able to limit the number of images displayed on your page while allowing the entire catalog for a user to be displayed in a fancybox lightbox? Here's what I have so far:

```
var feed = new Instafeed({
		get: 'user',
        userId: ****,
        accessToken: 'myAccessTolken',
		resolution: 'standard_resolution',
        limit: 4,
		template: '<a class="fancybox" href="{{model.images.standard_resolution.url}}"><img class="instagram-image" src="{{image}}" /></a>'
    });
    feed.run();
```
Any suggestions would be most appreciated. Thank you.

**Comment by garand** [comment]
You could remove the limit and hide anything after the 4th one with CSS.

**Comment by gbrandallen** [comment]
Thanks for the suggestion and I thought about that, however, my site is responsive (and has a width of 100%), as the window changes so does the height of that row. I'm not sure how I would be able to calculate the ever-changing height. I'm not opposed to your suggestion, I'm just not sure how I'd make that happen. Any further ideas would be appreciated.

**Comment by garand** [comment]
Can I have a link to your site? I might be able to put something together that works.

**Comment by gbrandallen** [comment]
Sure here you go. I'm super curious how you would tackle this. Thanks for the assistance.

http://new.signsplusnw.com/

**Comment by garand** [comment]
Thanks, can you tell me what exactly you're looking to have happen?

**Comment by gbrandallen** [comment]
What I was trying to do is display a feed of the 4 most recent photos on the homepage, but when you selects a photo a lightbox opens up (I use fancyBox) and the user is able to view a larger feed, maybe up to 20 photos. Make sense?

**Comment by gbrandallen** [comment]
Any luck?

**Comment by gbrandallen** [comment]
Nevermind, I figured it out. My solution is quick and dirty but it works effectively. I just displayed 20 images using the "limit" option in the initiator script, and used display:none; on the child selectors to hide the 16 images I didn't want displayed on the page. Fancybox still recognizes them and pulls them in when initiated. Works like a charm. I just the better solution would be to display them in an array...but that was a little past my pay grade.
